Sources/Everglow.Function/VFX/VFXManager.cs

```diff
@@ -38,7 +38,7 @@ public VFXManager(IHookManager hookManager, IMainThreadContext mainThread)
 	{
 		foreach (var layer in drawLayers)
 		{
-			visuals[layer] = new List<IVisualCollection>();
+			visuals[layer] = [];
```

Comment:
Error CS1525: Invalid expression term '['	in Everglow.Function	Line 41
Error CS0443: Syntax error; value expected	in Everglow.Function	Line 41